includes/Modules/Optimize.php

```diff
@@ -152,7 +160,7 @@ protected function print_amp_optimize_experiment() {
 		?>
 		<amp-experiment>
 			<script type="application/json">
-				<?php echo wp_json_encode( $optimize_option['ampExperimentJSON'] ); ?>
```

Comment:
There isn't very much context visible here but the `$optimize_option` variable doesn't exist. `$amp_experiment_json` is a json string.